src/main/java/com/hedera/exchange/ExchangeRateUtils.java

```diff
@@ -173,4 +175,132 @@ public static String getDecryptedValueFromAWS(final String value, final String l
 		}
 		return  nodes;
 	}
+
+	/**
+	 * Converts the string - string mapping of node id and address in the addressbook to
+	 * AccountID - string map so that, it can be used in hedera client directly.
+	 * @return
+	 */
+	public static Map<String, AccountId> getNodesForClient(Map<String, String> nodes) {
+		final Map<String, AccountId> accountToNodeAddresses = new HashMap<>();
+		for (final Map.Entry<String, String> node : nodes.entrySet()) {
+			final AccountId nodeId = AccountId.fromString(node.getKey());
+			accountToNodeAddresses.put(node.getValue(), nodeId);
+		}
+		return accountToNodeAddresses;
+	}
+
+	/**
+	 * Get the EPOC time of the end of the current hour in seconds in UTC.
+	 * for example, say the current date and time is October 31st 2019, 10:34 AM
+	 * then currentExpirationTime will be 1572537600 in UTC
+	 * @return
+	 */
+	public static long getCurrentExpirationTime() {
+		final long currentTime = System.currentTimeMillis();
+		final long currentHourOnTheDot = currentTime - (currentTime % 3_600_000);
```

Comment:
it looks like SonarQube is not running on this branche. Assign `3_600_000` to a constant with a very descriptive name